web/pages/profile.tsx

```diff
@@ -0,0 +1,595 @@
+import {
+  Button,
+  Flex,
+  Stack,
+  Text,
+  IconButton,
+  Input,
+  Box,
+} from "@chakra-ui/react";
+
+import { EditIcon } from "@chakra-ui/icons";
+
+import React from "react";
+import Select from "react-select";
+import {
+  Age,
+  Behavioral,
+  Breed,
+  FosterType,
+  Gender,
+  GoodWith,
+  Size,
+  Status,
+  Temperament,
+} from "../utils/types/post";
+
+export default function Profile() {
+  const [editing, setEditing] = React.useState(false);
+
+  const fosterTypes = [
+    { value: FosterType.Return, label: "Return" },
+    { value: FosterType.Boarding, label: "Boarding" },
+    { value: FosterType.Temporary, label: "Temporary" },
+    { value: FosterType.FosterMove, label: "Foster Move" },
+    { value: FosterType.Shelter, label: "Shelter" },
```

Comment:
Add an option for owner surrender according to the `FosterType` enum. I think these designs were created before we discussed these options with the nonprofit.